students/TimPauley/session02/fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+#Tim Pauley
+#Assignment 02
+#Date due: 10/23/2018
+
+
+def fizzbuzz(f):
+	for i in range(1, 101):
+    		options_to_print = [i, 'Fizz', 'Buzz', 'FizzBuzz']
```

Comment:
very slick solution -- but it would be good to define  "options_to_print"  outside the loop -- they don't change.